readme.md

```diff
@@ -82,10 +121,11 @@ Size of the returned icon.
 #### destination
 
 *Required*<br>
-Type: `string`
-
```

Comment:
Have you removed this new line by mistake?

Reply:
Probably